src/commands.ts

```diff
@@ -428,6 +430,43 @@ function openInExternalEditor(packageNode: PackageNode) {
     }
 }
 
+interface IDawrinCompatibleTarget extends vscode.QuickPickItem {
+    value: DarwinCompatibleTarget;
+    label: string;
+}
+
+async function handleQuickSwitch() {
```

Comment:
QuickSwitch is a very vague function name, SwitchPlatform maybe

Reply:
Yeah, that's entirely my bad 😓. I'll rename this based off your suggestion and add some documentation.